helm/prometheus-rules/templates/alerting-rules/cilium.rules.yml

```diff
@@ -33,3 +33,18 @@ spec:
         severity: page
         team: cabbage
         topic: cilium
+    - alert: CiliumFailedNetworkPolicy
+      annotations:
+        description: '{{`Too many Cilium Network Policy errors.`}}'
+        opsrecipe: unsupported-cilium-network-policy/
+      # cilium_policy_change_total - for cilium >=1.15
+      # cilium_policy_import_errors_total - for cilium <1.15
+      expr: max(rate(cilium_policy_change_total{outcome=~"fail.*"}[20m]) OR rate(cilium_policy_import_errors_total[20m]) OR vector(0)) > 0
```

Comment:
Is Vector(0) supposed to replace absent metrics ? I'm not sure this will work correctly because this Vector will have no labels. What is your intent?

Reply:
> Is Vector(0) supposed to replace absent metrics ?

Yes, I'll remove and re-run unit tests to see. I guess it should work without it.